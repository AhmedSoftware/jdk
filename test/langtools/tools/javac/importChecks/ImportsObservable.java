/*
 * @test /nodynamiccopyright/
 * @bug 4869999 8193302
 * @summary Verify that the compiler does not prematurely decide a package is not observable.
 * @compile --release 8 ImportsObservable.java
 * @compile -J-XX:+UnlockExperimentalVMOptions -J-XX:hashCode=2 --release 8 ImportsObservable.java
 * @compile/fail/ref=ImportsObservable.out -XDrawDiagnostics ImportsObservable.java
 */

import javax.*;
import javax.swing.*;
public class ImportsObservable {
}
