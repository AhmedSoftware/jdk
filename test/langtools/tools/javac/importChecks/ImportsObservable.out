ImportsObservable.java:10:8: compiler.err.doesnt.exist: javax
1 error
